pkg/square/square_fuzz_test.go

```diff
@@ -52,7 +52,7 @@ func FuzzSquare(f *testing.F) {
 		builder, err := square.NewBuilder(appconsts.DefaultMaxSquareSize, orderedTxs...)
 		require.NoError(t, err)
 		totalPfbs := builder.NumPFBs()
-		totalNormalTxs := builder.NumTxs()
+		totalNormalTxs := builder.NumTxs() - totalPfbs
```

Comment:
[optional] an exported function on the builder for normal txs: `builder.NumNormalTxs()`

Reply:
I think I'll wait for the need to arise first